docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3'
+
+services:
+  home-assistant:
+    image: homeassistant/home-assistant
+    container_name: home-assistant
```

Comment:
In docker compose, container names are usually not needed. I think we should remove this option to keep this file as simple as possible so that new users can understand it better.

Reply:
The only thing I would say for this one is that removing the `container_name` can create odd names like `home-assistant_home-assistant_1` in my case for example. Explicitly setting this to `home-assistant` makes it easier to access logs or restart containers by running `docker restart home-assistant` for example.

Reply:
Well when you're alreadY in the docker compose ecosystem like here you can also just run `docker-compose restart home-assistant`.

This also makes viewing logs much easier `docker-compose logs -f home-assistant`

Reply:
Oh, I didn't know that. OK I'll remove the `container-name`